src/java/org/apache/cassandra/utils/PojoToString.java

```diff
@@ -120,7 +120,7 @@ public static String pojoMapToString(Map<String, Object> map, String formatStrin
         return pojoToString(map, formatString);
     }
 
-    private static String pojoToString(Object obj, String formatString)
+    public static String pojoToString(Object obj, String formatString)
```

Comment:
not for this patch: why isn't the second argument `Format`?

Reply:
I'm guessing it's because it starts as a string in `listConsensusMigrations()` and we just didn't convert earlier. For my patch, it looks like I may actually be able to avoid `PojoToString` altogether, looking at the comments above...